### PR TITLE
Dice with Death Game  Issue#34

### PR DESCRIPTION
**Pull request by devika-singh**
# Pull Request for Play With Python 💡

## Issue Title : Dice with Death Game

- **Info about the related issue (Aim of the project)** : Implement the Dice with Death Game using Python
- **Name:** Devika Singh
- **GitHub ID:** devika-singh
- **Email ID:** devikas.iitg@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in.)** CODEPEAK 2022

Closes: #34 

### Describe the add-ons or changes you've made 📃
I have added the game Dice with Death folder and its game itself, with main.py, Images folder and images of game screens and the README.md
Used https://www.youtube.com/watch?v=ex24RomyPXk to understand rules of the game.

## Type of change ☑️

What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
I entered yes in continue playing till all test cases had been encountered

## Checklist: ☑️
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Can you upgrade the user interaction technique from CLI to GUI. It'll be better for the users to have more interaction with the game.
@devika-singh

**Comment by devika-singh** [comment]
@abhisheks008  I made the changes pls re-review it

File: Dice with Death/main.pyw [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# importing the random module
```
Comment (abhisheks008) [inline Dice with Death/main.pyw:1]: The extension will be, `.py` instead of `.pyw`.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Review by abhisheks008 — APPROVED** [review]
Your PR is approved and ready to be merged.
@devika-singh

**Pull request merged**